### PR TITLE
Soback 87 broken icons in toolbar

### PR DESCRIPTION
**Pull request by jesperlauridsen**
Added icons and did a little polish on the toolbar, css-wise.

### CODE REVIEW

**Review by jorntx — APPROVED** [review]
Works and nice touch up!

Approved - waiting for @thomasegense to approve before I hit merge

**Comment by thomasegense** [comment]
Just needs a new icon for PWID xml. ( I suggest the text_snippet icon)

**Review by jorntx — APPROVED** [review]

**Pull request merged**
